scripts/select_strains.py

```diff
@@ -177,6 +181,19 @@ def parse_metadata(segments, metadata_files):
         metadata[segment] = tmp_meta
     return metadata
 
+def parse_sequences(segments, sequence_files):
+    """Load sequence names into a dictionary of sets indexed by segment.
+    """
+    sequences = {}
+    for segment, filename in zip(segments, sequence_files):
+        sequence_set = Bio.SeqIO.parse(filename, "fasta")
+        sequences[segment] = set()
```

Comment:
wouldn't:
sequences[segment] = set(Bio.SeqIO.parse(filename, "fasta"))

do the trick? Does it matter with list of set?

Reply:
sorry, I misread and didn't realize that you only glob the names. still a list comprehension would do.